specs/market-api.yaml

```diff
@@ -981,57 +987,76 @@ components:
       allOf:
         - $ref: '#/components/schemas/DemandOfferBase'
         - type: object
-          properties:
-            demandId:
-              type: string
-              readOnly: true
-            requestorId:
-              type: string
-              readOnly: true
+      properties:
+        timestamp:
+          type: string
+          format: date-time
+        demandId:
+          type: string
+          readOnly: true
+        requestorId:
+          type: string
+          readOnly: true
+      required:
+        - demandId
+        - requestorId
 
     Offer:
       allOf:
         - $ref: '#/components/schemas/DemandOfferBase'
         - type: object
-          properties:
-            offerId:
-              type: string
-              readOnly: true
-            providerId:
-              type: string
-              readOnly: true
+      properties:
```

Comment:
this dedent causes openapi generator not to generate fields below
the same for other dedents